schema/config-linux.json

```diff
@@ -245,6 +245,10 @@
                                 "id": "https://opencontainers.org/schema/bundle/linux/resources/memory/kernel",
                                 "$ref": "defs.json#/definitions/int64Pointer"
                             },
+                            "kernelTCP": {
+                                "id": "https://opencontainers.org/schema/bundle/linux/resources/memory/kernelTCP",
+                                "$ref": "defs.json#/definitions/int64Pointer"
```

Comment:
We definitely need a `kernelTCP` entry (catching up with #235), but after #555 I'm not sure we want to use pointers here.  The optional-ness is represented in JSON Schema by not listing these properties in a `required` array; we don't have to (or want to?) allow people to explicitly set a `null` value.